scripts/ceph-windows/collect-event-logs.ps1

```diff
@@ -5,7 +5,7 @@ param (
     [switch]$CleanupEventLog = $false
 )
 
-$ErrorActionPreference = "Stop"
+$ErrorActionPreference = "Ignore"
 
 function DumpEventLogEvtx($path){
     foreach ($i in (Get-WinEvent -ListLog * |  ? {$_.RecordCount -gt 0 })) {
```

Comment:
Shouldn't the condition `$_.RecordCount -gt 0` from here make sure that this doesn't happen:
```
The "Get-WinEvent" command used to retrieve Windows event log messages can fail if the specified log has no entries.
```

Reply:
There's an additional filter by timestamp:

```
Get-WinEvent `
            -ErrorAction "Ignore" `
            -FilterHashtable @{
                LogName=$i.LogName;
                StartTime=$(Get-Date).AddHours(-6)
```

So it could be that there's a record that occurred before the specified timeframe, in which case it errors out.

Reply:
Fair enough!